RGW overview dashboard update

s0nea opened the pull request:
- Updates the ceph-rgw-overview dashboard to the latest version.
- Remove ceph-rgw-workload dashboard because the content has been moved to ceph-rgw-overview.

richardtief reviewed (changes requested):
Please upgrade the version in `Chart.yaml` and both versions in `plugindefinition.yaml`. Then it's ready to go.

s0nea commented:
> Please upgrade the version in `Chart.yaml` and both versions in `plugindefinition.yaml`. Then it's ready to go.

Thank you for your review @richardtief. I addressed your comment. Could you please check it again?

richardtief reviewed (approved):
Thanks.
